app/Models/UserDetail.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App\model;
```

Comment:
same case with namespace here, `namespace App\Models` (follow case of the folder)